docs/inso-cli/cli-command-reference/OAS-spec.md

```diff
@@ -0,0 +1,615 @@
+---
+layout: article-detail
```

Comment:
@lena-larionova I added this new page in reference to this slack conversation: https://kongstrong.slack.com/archives/CLDMTEH5M/p1650378201802369?thread_ts=1650356318.998349&cid=CLDMTEH5M

Reply:
I don't know too much about how the Insomnia site is built, but from general experience - you don't want to upload this as a markdown file. Mainly, that's because it's not meant to be one - take a look at how it renders: https://insomnia-docs-git-ga-04-21-teaminsomnia.vercel.app/inso-cli/cli-command-reference/OAS-spec

I suggest uploading the yaml file instead of turning it into markdown and linking it from the navigation. So an entry for the spec would go here:

![Screen Shot 2022-04-21 at 9 20 47 AM](https://user-images.githubusercontent.com/54370747/164506471-b9df7d0a-5391-4492-8c28-79faeba1caa3.png)

Reply:
I would also suggest a title of "OpenAPI Spec Reference" (title case to be consistent).

Reply:
Edit this nav file:
https://github.com/Kong/insomnia-docs/blob/GA-04-21/docs/_data/main-nav.yaml#L144

After line 144, add en entry for the yaml link:
```
- title: OpenAPI Spec Reference
   url: /inso-cli/cli-command-reference/OAS-spec.yml
```

Reply:
THANKS!!!

Reply:
It works!